docker/docker-compose-core.yml

```
@@ -4,7 +4,7 @@ services:
   # ICUBAM service
   app-server:
     build:
-      context: "${ICUBAM_COMPOSE_CONTEXT:-.}"
+      context: "${ICUBAM_COMPOSE_CONTEXT:-..}"
```

See comment below about the mount
```suggestion
      context: "${ICUBAM_COMPOSE_CONTEXT:-.}"
```